_maps/RandomZLevels/undergroundoutpost45.dmm

```diff
@@ -3614,7 +3614,7 @@
 	},
 /area/awaymission/undergroundoutpost45/research)
 "hD" = (
-/obj/machinery/rnd/production/protolathe,
+/obj/machinery/rnd/production/protolathe/offstation,
```

Comment:
no reason to bother here? its an away mission, accessible only from the gateway which is on the station, without any ghost roles i imagine

Reply:
I just did a find-and-replace on all protolathes, since no non-departmental variants were onstation. But I can change it back, since you're right that it is just an away mission.

Reply:
Restored.